Godeps/_workspace/src/github.com/openshift/source-to-image/pkg/build/strategies/sti/sti.go

```diff
@@ -143,7 +143,11 @@ func (b *STI) Build(config *api.Config) (*api.Result, error) {
 	}
 
 	if b.incremental = b.artifacts.Exists(config); b.incremental {
-		glog.V(1).Infof("Existing image for tag %s detected for incremental build", config.Tag)
+		tag := config.IncrementalFromTag
+		if len(tag) == 0 {
+			tag = config.Tag
+		}
+		glog.V(1).Infof("Existing image for tag %s detected for incremental build", tag)
```

Comment:
These are the interesting bits, the rest is just the removal of test files added in the last bump by @csrwng.
The [script in source-to-image](https://github.com/openshift/source-to-image/blob/master/hack/rebase-to-origin.sh) explicitly remove test files, godep doesn't, that's the difference.
Talked with @csrwng and concluded we don't require the test files under Godeps/_workspace.